Base/QTCore/qSlicerCoreApplication.cxx

```diff
@@ -1872,3 +1896,85 @@ void qSlicerCoreApplication::showConsoleMessage(QString message, bool error/*=tr
     std::cout << message.toLocal8Bit().constData() << std::endl;
   }
 }
+
+// --------------------------------------------------------------------------
+QString qSlicerCoreApplication::toSlicerHomeAbsolutePath(const QString& path) const
+{
+  Q_D(const qSlicerCoreApplication);
+  if (path.isEmpty())
+    {
+    return path;
+    }
+  QFileInfo fileInfo(path);
+  if (fileInfo.isRelative())
```

Comment:
That is clever, that way there is no need for any special placeholder and it is always relative to Slicer home :+1: